utils.sh

```diff
@@ -552,7 +552,7 @@ function bootstrap_ip {
                       | grep "${pref_ip}" \
                       | tail -n1 \
                       | awk '{print $5}' \
-                      | sed -e 's/\(.*\)\/.*/\1/'
+                      | sed -e 's/\(.*\)\/.*/\1/' || true
```

Comment:
If we're doing this we could arguably remove the DHCP_LEASE_EXPIRY check too. We may actually have a new enough libvirt that we don't need to skip this anymore.

Reply:
Sounds like we could handle that as a followup since it's not strictly related to this change?